### ISSUE TITLE
Null Object Reference crash trying to set keyboard shortcut

### ISSUE THREAD

**Issue by IBwWG** [larsiusprime/tdrpg-bugs#688]
In 0.5.2 full, win7x64.

Just clicked the "cycle defenders" "," after selecting Dvorak layout, but it's also repeatable without that step.

```
Called from ApplicationMain::main ApplicationMain.hx line 159
Called from ApplicationMain::create ApplicationMain.hx line 71
Called from lime.app.Application::exec lime/app/Application.hx line 222
Called from lime._backend.native.NativeApplication::exec lime/_backend/native/Na
tiveApplication.hx line 136
Called from lime._backend.native.NativeApplication::handleRenderEvent lime/_back
end/native/NativeApplication.hx line 368
Called from lime.app.Event_Void_Void::dispatch lime/app/Event.hx line 145
Called from *::_Function_2_1 lime/app/Application.hx line 110
Called from lime.app.Application::render lime/app/Application.hx line 762
Called from openfl.display.Stage::render openfl/display/Stage.hx line 677
Called from openfl.display.DisplayObjectContainer::__broadcast openfl/display/Di
splayObjectContainer.hx line 355
Called from openfl.display.DisplayObject::__broadcast openfl/display/DisplayObje
ct.hx line 251
Called from openfl.events.EventDispatcher::__dispatchEvent openfl/events/EventDi
spatcher.hx line 237
Called from flixel.FlxGame::onEnterFrame flixel/FlxGame.hx line 529
Called from flixel.FlxGame::step flixel/FlxGame.hx line 662
Called from com.leveluplabs.tdrpg.Main::update com/leveluplabs/tdrpg/Main.hx lin
e 1462
Called from flixel.FlxGame::update flixel/FlxGame.hx line 751
Called from flixel.FlxState::tryUpdate flixel/FlxState.hx line 178
Called from flixel.FlxState::tryUpdate flixel/FlxState.hx line 178
Called from flixel.FlxState::tryUpdate flixel/FlxState.hx line 168
Called from com.leveluplabs.tdrpg.Popup_Options_SubMenu_Controls::update com/lev
eluplabs/tdrpg/Popup_Options_SubMenu_Controls.hx line 96
Called from com.leveluplabs.tdrpg.UI_KeyButtonManager::update com/leveluplabs/td
rpg/UI_KeyButtonManager.hx line 118
AL lib: (EE) alc_cleanup: 1 device not closed
```

**Comment by larsiusprime** [comment]
I can't seem to reproduce this, but that doesn't mean it isn't happening. If you can still reproduce it consistently, can you take a screenshot of the controls menu right before you click on it? Maybe something is subtly different in your setup.

**Comment by IBwWG** [comment]
Yep, had it happen 3 times I think.  I'll take a screenshot soon.

**Comment by IBwWG** [comment]
![dq-crash-setting-key](https://cloud.githubusercontent.com/assets/16719964/14833474/d4d0f90e-0bff-11e6-96d5-3dc94f65af48.png)
(on 0.5.3 win7x64)

**Comment by larsiusprime** [comment]
Okay I see it now, this is just (another!) simple null check I need.

**Comment by larsiusprime** [comment]
Presumed fixed in 0.5.4
